notebooks/kubeflow_pipelines/walkthrough/solutions/kfp_walkthrough_vertex_pytorch.ipynb

```diff
@@ -0,0 +1,1517 @@
+{
```

Comment:
<p>Ideally, <code>FeatureTransformerBase</code> be should  be an abstract class, since it is not meant to be instantiated (see https://docs.python.org/3/library/abc.html)</p><p> </p><p>Do we really need to reimplement these transformations?  Doesn't Pytorch has a module that does that? If so, we should re-use these here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>Yeah I totally agree, FeatureTransformerBase shouldn't be used directly. </p><p> </p><p>In terms of pre-defined functions, I also searched it, but there seems to be no handy and well-wrapped modules for it. </p><p>That's why I implemented it (and wrote it as reusable as possible)!</p>